core/os/condition_variable.h

```diff
@@ -31,6 +31,9 @@
 #ifndef CONDITION_VARIABLE_H
 #define CONDITION_VARIABLE_H
 
+#include "core/os/mutex.h"
+#include "core/typedefs.h"
```

Comment:
Adding these includes seems correct, I wonder how it compiled before? I guess it just happened to be always included after `mutex.h`?